.gitignore

```diff
@@ -108,3 +108,6 @@ venv.bak/
 
 #IntelliJ
 .idea
+
+# Visual Studio Code
+.vscode
```

Comment:
add newline at end of file

Reply:
Addressed via a9e561634873616e2cae62ee6d8667fdd1ef8431

Reply:
And subsequently fixed via a558940a99bb0559b3c6f56e8c36c6d91b112985